src/components/Camera/index.js

```diff
@@ -76,10 +78,10 @@ export default class Camera extends Component {
     const { method, onUserMedia } = this.props
     return (
       <div>
-        <div className='onfido-video-overlay'>
+        <div className={style["video-overlay"]}>
```

Comment:
Can't really be spotted by the linter though.. Also the quotes will not be passed as a value to the JSX attribute.
`<Foo style={{ left: "20px" }} />`, the quotes in this example are actually passed as an attribute, however the result of `style["video-overlay"]` is what will be passed as a JSX attribute.